oap-server/server-starter/src/main/resources/ui-initialized-templates/bookkeeper/bookkeeper-cluster.json

```diff
@@ -0,0 +1,393 @@
+[
+  {
+    "id": "Bookkeeper-Cluster",
```

Comment:
`Bookkeeper` -> `BookKeeper`

Please keep the name correctly in all things of your PR.